CONTRIBUTING.md

```diff
@@ -69,7 +69,7 @@ plus various other directories with `regression-tests.*` names.
 ## Commit Guidelines
 * Tell why the change does what it does, not how it does it.
 * The first line should be short (preferably less than 50 characters)
-* The rest of the commit body should be wrapped at 72 characters (see [this](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more info)
+* The rest of the commit body should be wrapped at 72 characters (see [this](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for more info)
```

Comment:
I'm mostly replacing `http:` with `https:` (with a few exceptions). I haven't made such changes where the urls don't work.

I haven't canonicalized, so if a domain switched from `www.example.com` to `example.com` or vice versa, I didn't harmonize them (there are cases where both are used in the codebase, I'll call that out later).